src/fsharp/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.nuspec

```diff
@@ -40,13 +39,16 @@
         </dependencies>
     </metadata>
     <files>
-        <file src="fsc.exe"                           target="lib/netstandard1.6" />
-        <file src="FSharp.Compiler.dll"               target="lib/netstandard1.6" />
-        <file src="default.win32manifest"             target="runtimes/any/native/" />
-        <file src="FSharp.Build.dll"                  target="lib/netstandard1.6" />
-        <file src="Microsoft.FSharp.targets"          target="runtimes/any/native/" />
-        <file src="Microsoft.Portable.FSharp.targets" target="runtimes/any/native/" />
-        <file src="Microsoft.FSharp.NetSdk.targets"   target="runtimes/any/native/" />
-        <file src="Microsoft.FSharp.NetSdk.props"     target="runtimes/any/native/" />
+        <file src="coreclr/bin/fsc.exe"                           target="lib/netstandard1.6" />
```

Comment:
@dsyme publish, doesn't really dig the tools folder.  I guess it's meant for tools you need when working with whatever is referenced.

The Roslyn compiler tools, put everything in the runtime/any/native folder with a marker file in the ref/netcoreapp1.0 folder.  Which makes a sort of cunning sense.

So that is what I will do too.